test/unit/user_test.rb

```diff
@@ -305,4 +305,7 @@ class UserTest < ActiveSupport::TestCase
     end
   end
 
+  test 'Send digest email for unmoderated posts' do
+    assert users(:bob).send_digest_email_spam
```

Comment:
Here, or in our functional tests, i wonder if we can check the content of the email itself to confirm? There are some other examples of this in our tests. It would help secure this feature against future issues!